docusaurus/docs/guides/database-lifecycle.mdx

```diff
@@ -0,0 +1,44 @@
+---
+title: Database Lifecycle Management
+sidebar_position: 30
+---
+
+The OpenZiti Controller uses a [Bolt](https://github.com/boltdb/bolt) database to store its state. The database is a writeable file in a path specified by the controller's configuration property `.db` ([link to reference](/reference/30-configuration/controller.md#db)).
+
+Use the controller's built-in database snapshot operation to preserve data integrity. The snapshot operation creates a snapshot file in the same filesystem directory as the main database file on the controller host.
+
+There are three ways to invoke the snapshot operation.
+
+1. Agent (IPC): run the `snapshot-db` command on the controller host.
```

Comment:
IMO, ziti would only have one way of donig this. i'd like to only see "use the ziti cli" here. I don't think it's necessary to add in the complexity of the agent here.) I feel like the agent method should only be a stopgap for when the system isn't responding to the rest API.

Reply:
I've trimmed it down to:

1. login then snapshot with CLI anywhere REST API is reachable
2. snapshot on controller host without login

It's unnecessary to redundantly document the REST operation because it's adequately covered by the spec and generated reference, which is now linked instead of explained.